### PR TITLE
feat: update eth2 staking stats

### PR DESCRIPTION
**Pull request by kelsos**
[skip ci]

Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/locales/en.json [hunk 1]
```diff
@@ -1451,6 +1451,7 @@
     "price_in_symbol": "Price in {symbol}",
     "priority": "Priority",
     "protocol": "Protocol",
+    "rewards": "@:common.rewards",
```
Comment (lukicenturi) [inline frontend/app/src/locales/en.json:1454]:
```
"rewards": "Rewards",
```

**Comment by lukicenturi** [comment]
image

from beginning, there's no `dailystats` that called with `only_cache=false`.
So the table always empty.
image

File: frontend/app/src/locales/en.json (hunk 1 above)
Comment (kelsos) [reply frontend/app/src/locales/en.json:1454]: I did mass replace everything here, didn't I?

**Comment by lukicenturi** [comment]
It seems that this endpoint `<POST> /blockchains/ETH2/stake/details` is always error when called with empty payload `({})`

image

But if we send the validator_index, it works
image

Not sure if this should be fixed in backend or frontend.

**Comment by lukicenturi** [comment]
> image
> 
> from beginning, there's no `dailystats` that called with `only_cache=false`. So the table always empty. image

You did the `only_cache=False`, but I think you forget to set the state from the task call.
I need to call once again with `only_cache=True` to see the table filled.

**Review by lukicenturi — APPROVED** [review]
LGTM!

**Pull request merged**